### PR TITLE
Improve word matching for kebab case.

### PR DESCRIPTION
**Pull request by floitsch**
Also add auto-indent.

### CODE REVIEW

File: vscode/language-configuration.json [hunk 1]
```diff
@@ -27,5 +27,10 @@
         ["(", ")"],
         ["\"", "\""],
         ["'", "'"]
-    ]
+    ],
+    "wordPattern": "(-?\\d*\\.\\d\\w*)|([^\\`\\~\\!\\@\\#\\%\\^\\&\\*\\(\\)\\=\\+\\[\\{\\]\\}\\\\\\|\\;\\:\\'\\\"\\,\\.\\<\\>\\/\\?\\s\\$]+)",
```
Comment (floitsch) [inline vscode/language-configuration.json:31]: This line comes from here: https://code.visualstudio.com/api/language-extensions/language-configuration-guide

I just removed the `\\-` and added a `\\$`.

File: vscode/language-configuration.json [hunk 2]
```diff
@@ -27,5 +27,10 @@
...
+    "indentationRules": {
+        "increaseIndentPattern": "[\\:]([\\s]*\\|(\\s*[\\w-]+)+\\s*\\|)?$",
```
Comment (floitsch) [inline vscode/language-configuration.json:33]: Automatically increase for trailing ':', but allow block parameters.

File: vscode/package.json [hunk 3]
```diff
@@ -17,15 +17,21 @@
...
 		"onView:toitView",
-		"onCommand:toit.ensureAuth",
 		"onCommand:toit.refreshView"
 	],
 	"icon": "images/toitware_logo.png",
 	"main": "./dist/extension.js",
 	"contributes": {
+		"configurationDefaults": {
+			"[toit]": {
+				"editor.wordSeparators": "`~!@#$%^&*()=+[{]}\\|;:'\",.<>/?",
```
Comment (floitsch) [inline vscode/package.json:29]: This line comes from here: https://github.com/microsoft/vscode/issues/87310

**Review by kasperl — APPROVED** [review]
Looks good to me.

**Pull request merged**